### ISSUE TITLE
icon sizes are not aligned with our size rhythms conventions

### ISSUE THREAD

**Issue by Kiarokh** [Lundalogik/lime-elements#1854]
## Current behavior
related to https://github.com/Lundalogik/lime-elements/issues/2741 

We follow a size rhythm according to [our design guidelines](https://lundalogik.github.io/lime-elements/versions/next/#/DesignGuidelines/size-rhythms.md/). However, our preset icon sizes (`x-small` > `small` > … > `large`) do not follow this guideline.

https://github.com/Lundalogik/lime-elements/blob/598ceb8aa6ac7c718fdda1a65aa477c116602c02/src/components/icon/icon.scss#L26

## Expected behavior
Icon sizes should also be equivalent to multiplies of `4px` as described in the guidelines.

However, since icons are used in other components, we need to make sure that changing sizes won't negatively affect the UI.

## Environment

- **lime-elements** version: 
- Framework used: 
- Logs:

**Comment by paulinewahle** [comment]
As mentioned in the debrief, following the rhythm by progressing in an 4x interval is not realistic for the icon sizes. One way to stay close to the rhythm in smaller intervals is to add 1/4 per increase. The resulting sizes are very close to the version before, so it shouldn’t create a mess where they are used.

**Comment by Kiarokh** [comment]
@paulinewahle let's have a talk about this. But in short, I remember that I tried to change this at some point. But since icon sizes are used in many other components, this issue become s really hard to solve. I myself gave up. Don't put time on this issue at all. We may come back to it at som point. But this is not a "good first issue".

**Comment by Kiarokh** [comment]
I am gonna approve your PR @paulinewahle 
I made small changes to it, and in a couple of places chose to render the icons a couple of pixels smaller than what they were before. This way, it's still safe and previously made UIs should behave the same, or at least not get totally destroyed.

I'm still not happy with what we have. Because:

In my head, it shouldn't matter whether there is a badge or not. The size of the icon element should not change, if there is a badge. However, this has been a choice since ages ago, before we stablished the size conventions.

Fixing it everywhere is really hard.

I'd like us to go remove all those `size="smething"` from all `limel-icon` instances everywhere in our code. I don't think we should use them at all ourselves. Sizes should be based on whatever the UI-design demands per case. 

Anyhow, this is a lot of work and we can do it some beautiful day, where nothing else matters. 😄 
For now, we can be happy that things align more perfectly, at least according to our sizing conventions.
